src/app/pages/ElectronicDelivery.jsx

```diff
@@ -296,6 +302,7 @@ const mapStateToProps = (state) => ({
   bib: state.bib,
   searchKeywords: state.searchKeywords,
   features: state.features,
+  isEddRequestable: state.isEddRequestable
```

Comment:
Don't forget to add new props (in this case, isEddRequestable) to the static propTypes object.